injected/src/features/duckplayer/components/ddg-video-drawer-mobile.js

```diff
@@ -166,7 +166,7 @@ export class DDGVideoDrawerMobile extends HTMLElement {
         this.overlay = /** @type {HTMLElement} */ (overlay);
         this.drawer = /** @type {HTMLElement} */ (drawer);
 
-        infoButton.addEventListener('click', () => {
+        infoButton.addEventListener('pointerdown', () => {
```

Comment:
The mobile-specific changes can be removed if this is only going out on Windows.